packages/manager-components/src/components/templates/base/base.component.tsx

```diff
@@ -2,41 +2,45 @@ import React from 'react';
 import { Headers, HeadersProps } from '../../content';
 import { Description, LinkType, Links, Subtitle } from '../../typography';
 import { PageLayout } from '../layout/layout.component';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
 
-export interface DashboardLayoutProps {
+export type BaseLayoutProps = React.PropsWithChildren<{
   breadcrumb?: React.ReactElement;
-  content?: React.ReactElement;
   header?: HeadersProps;
   message?: React.ReactElement;
   description?: string;
   subtitle?: string;
   subdescription?: string;
```

Comment:
**question:** Can we change this to `subDescription`?

Reply:
👍